extension/export_util/export_hf_model.py

```diff
@@ -74,11 +74,11 @@ def main() -> None:
     print(f"{model.generation_config}")
 
     tokenizer = AutoTokenizer.from_pretrained(args.hf_model_repo)
-    input_ids = tokenizer([""], return_tensors="pt").to(device)["input_ids"]
+    input_ids = torch.tensor([[1]], dtype=torch.long)
     cache_position = torch.tensor([0], dtype=torch.long)
 
     def _get_constant_methods(model: PreTrainedModel):
-        return {
+        metadata = {
             "get_dtype": 5 if model.config.torch_dtype == torch.float16 else 6,
             "get_bos_id": model.config.bos_token_id,
             "get_eos_id": model.config.eos_token_id,
```

Comment:
@larryliu0820 Is it safe to leave it unset and let the runtime to use the default value? Or it's preferred to set it to some special value e.g. -1, if any of the field is unset by a model?